content/stub.compose-ui.js

```diff
@@ -684,8 +684,9 @@ ComposeSession.prototype = {
       draft: function ({ to, cc, bcc }) {
         let makeTokens = function (aList) {
           let [list, listEmailAddresses] = parse(aList);
-          return [asToken(null, item, listEmailAddresses[i], null)
-            for each ([i, item] in Iterator(list))];
+	  return Array.prototype.map.call(list, function(item, i) {
```

Comment:
You seem to hesitate between rewriting `[f(x, i) for each ([i, x] in Iterator(someArray))]` into either:

- `someArray.map(function (x, i) f(x, i))` or
- `[f(x, i) for ([i, x] of someArray)]`.

Since https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols#The_iterable_protocol tells us that `Array` implements the iteration protocol, the two above should be equivalent. I'd rather go for the former, what do you think?